models/gyroscope/gyroscope_trades.sql

```diff
@@ -1,7 +1,11 @@
 {{ config(
     schema = 'gyroscope',
     alias = 'trades',
-    materialized = 'view'
+    materialized = 'view',
+    post_hook='{{ expose_spells(\'["arbitrum", "ethereum","optimism", "polygon"]\',
```

Comment:
we typically only expose the cross-chain spell, assuming this is the one you'd query most on the app?

if so, i'd remove the post hook from the chain-specific spells

Reply:
sounds good! just expose the cross-chain spell should be enough

thanks!

Reply:
please do go ahead and push a commit to remove the chain-specific ones 🙏